CONTRIBUTING.md

```diff
@@ -64,12 +64,16 @@ The core team is monitoring for Pull Requests. We will review your Pull Request
 
 ### How to increase the chance of being accepted?
 
+We run a series of checks automatically when a PR is opened (CI). If you're not
```

Comment:
x4

```suggestion
We run a series of checks automatically when a Pull Request is opened (CI). If you're not
```

We don't use the acronym in the document, at least, not yet. I propose that we stick to the long-form. I have seen some people ask: what's a PR? a press release? 😆